kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/jaxb/JaxbCommandsResponse.java

```diff
@@ -212,8 +218,10 @@ public void addResult(Object result, int i, Command<?> cmd) {
             this.responses.add(new JaxbVariableInstanceLog((VariableInstanceLog) result));
         } else if( result instanceof DefaultFactHandle ) { 
            this.responses.add(new JaxbOtherResponse(result, i, cmd));
-        } else if( cmd instanceof GetTaskContentCommand || cmd instanceof GetContentMapForUserCommand ) { 
-           this.responses.add(new JaxbTaskContentResponse((Map<String, Object>) result, i, cmd));
+        } else if( cmd instanceof GetTaskContentCommand || cmd instanceof GetContentMapForUserCommand ) {
+            this.responses.add(new JaxbTaskContentResponse((Map<String, Object>) result, i, cmd));
+        } else if ( cmd instanceof GetTaskByWorkItemIdCommand ) {
+            this.responses.add(new JaxbTaskResponse((Task)result, i,  cmd));
```

Comment:
@psiroky I think it might be better to modify line 165 so that it gets added there? 

Also, I would make sure that the logger message shows the entire name, instead of just the simple name of the class.